v63004/instream.csh

```diff
@@ -2,33 +2,34 @@
 #################################################################
 #								#
 # Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
-# All rights reserved.						#
+# all rights reserved.						#
 #								#
 #	This source code contains the intellectual property	#
 #	of its copyright holder(s), and is made available	#
-#	under a license.  If you do not know the terms of	#
+#	under a license. If you do not know the terms of	#
 #	the license, please stop and do not read further.	#
 #								#
 #################################################################
 #
 #-----------------------------------------------------------------------------------------------------------------------------
-# List of subtests of the form "subtestname [author] description"
+# list of subtests of the form "subtestname [author] description"
 #-----------------------------------------------------------------------------------------------------------------------------
-# gtm8909	    [jake]  Tests that <ctrl-c> within the help facility no longer leads to EN0256 error upon exit
-# gtm8860	    [jake]  Tests that journal extract removes additional / from journal and output file paths
-# gtm8791	    [jake]  Tests that <ctrl-z> no longer causes segmentation violation
-# gtm8699	    [jake]  Tests that $VIEW("STATSHARE",<region>) returns 1 if the process is sharing DB stats and 0 otherwise
-# gtm8202	    [jake]  Tests the functionality of the -SEQNO qualifier for the mupip journal -extract command
-# gtm5730	    [jake]  Tests that the update process now logs record types with a corresponding, non-numerical, description
-# gtm1041	    [jake]  Tests the that env variable gtm_mstack_size sets the size of the M stack as expected
+# gtm8909	    [jake]  tests that <ctrl-c> within the help facility no longer leads to en0256 error upon exit
```

Comment:
I see a huge diff block here. I was expecting just the one line corresponding to the newly added subtest. What was changed in the other lines?

Reply:
The tests are listed in reverse numeric order. Since 8891 is second from the top, all the lines below it are displaced by 1 line which causes these changes

Reply:
The displacement of lines is not the issue. There have been a lot of upper case letters changed to lower case. That is the reason for the huge block diff and needs to be reverted to their original case.